language-plutus-core/stdlib/Language/PlutusCore/StdLib/Data/Function.hs

```diff
@@ -302,8 +315,8 @@ fixN n = runQuote $ do
 -- >                 \(fN1 : a1 -> b1) ... (fNn : an -> bn) -> choose f1 ... fn
 getMutualFixOf
     :: (TermLike term TyName Name, Functor term)
-    => ann -> [FunctionDef term TyName Name ann] -> Quote (Tuple term ann)
-getMutualFixOf ann funs = do
+    => ann -> term ann -> [FunctionDef term TyName Name ann] -> Quote (Tuple term ann)
+getMutualFixOf ann fixn funs = do
```

Comment:
Do you use `fix` instead of `fixN 1` by the way? I think the former should be more optimal.

Reply:
No, it would require changing the plumbing a bit. Good idea, though.